### ISSUE TITLE
Crash problem on Windows

### ISSUE THREAD

**Issue by Nik96i** [media-kit/media-kit#506]
Hi.
I use media_kit Player in my app. I'm developing on Windows 11 and the app crashes without any error. I also built the app and tested it again, same thing happened. it works fine on other PCs running Windows 11 but it has a problem with my PC.
DirectX and VC Redistributable are also installed.
Thanks.

**Comment by alexmercerind** [comment]
Hi @Nik96i,

We did get one similar bug-report few months ago. They had exact same issue on Windows. They ended up re-installing Windows, which fixed it for them. We couldn't find any cause or sufficient information at the time. We only received one such report, other users have no complaints.

You can try to run our demo application (direct executable available in README) & check if that works. A stack trace will be further helpful.

Thanks!

**Comment by Nik96i** [comment]
@alexmercerind Thanks u for ur reply sir. I can Replace my Windows but it may happen in one of our customer's systems, so it would be awesome if this problem resolves. We Cannot release our app like that. I tested the demo app and it also crashed. If u can provide a simple debug/test app which outputs every log or something, I can test it and maybe it provide some hints.
Here I tested the demo app:
[https://file.io/P1v8muczlE28](url)
Thanks again.

**Comment by alexmercerind** [comment]
Hi @Nik96i,

Does switching to software rendering work? Click the "H/W" button on the top-right of the application to make it "S/W".

**Comment by Nik96i** [comment]
@alexmercerind No. it doesn't work.

**Comment by Nik96i** [comment]
Hi @alexmercerind 

This is my crash log in the Windows event viewer.
I think the program cannot load "libmpv-2.dll" file.

```
Faulting application name: media_kit_test.exe, version: 1.0.0.0, time stamp: 0x6514abf2
Faulting module name: libmpv-2.dll, version: 2.0.0.0, time stamp: 0x650fea09
Exception code: 0xc0000094
Fault offset: 0x0000000000134ec4
Faulting process id: 0x0x166C
Faulting application start time: 0x0x1D9F374589E9231
Faulting application path: C:\Users\nik96i\Downloads\media_kit_test_win32_x64\buildwindowsrunnerRelease\media_kit_test.exe
Faulting module path: C:\Users\nik96i\Downloads\media_kit_test_win32_x64\buildwindowsrunnerRelease\libmpv-2.dll
Report Id: e203d615-2c4e-4c6a-a0b8-836902a09c1b
Faulting package full name: 
Faulting package-relative application ID: 
```

**Comment by aslanonurcan** [comment]
I got similar issue. Then I upgrade flutter and dart sdk to latest version and packages of course, now it works.

**Comment by Nik96i** [comment]
Hi @alexmercerind
I solved it by removing mpv-1.dll from my path. I had libmpv-2.dll and mpv-1.dll (mpvnet-player) in my path but didn't notice. so after removing mpv-1.dll, it works.

For every one that struggles with this issue, you can find the .dll path with this command:
```
where mpv-1.dll
```

**Comment by srix55** [comment]
Issue can be closed @alexmercerind

**Comment by alexmercerind** [comment]
This should be resolved in #619.
